fcrepo-exts/fcrepo-camel-toolbox
# Docker files for camel-toolbox

**JIRA Ticket**: https://fedora-repository.atlassian.net/browse/FCREPO-3746

# What does this Pull Request do?
Adds a Dockerfile for building an image with the camel-toolbox app as well as a docker-compose configuration file for running the camel-toolbox alongside Fedora and other components.

# What's new?

* Added Dockerfile to build camel-toolbox image
* Added Docker Compose configuration for running camel-toolbox

# How should this be tested?

Run with docker-compose (not 100% functional yet -- see notes):
```
docker-compose up -d
```

# Additional Notes:

Looking for a little bit of feedback on the docker files:
* General organization of the docker files
* Volumes/environment variables we might want to expose
* If we should build and deploy the camel-toolbox image to Dockerhub and pull that using compose rather than having it build the image (maybe provide instructions if people are testing changes

Also the network setup doesn't exactly seem compatible with the camel-toolbox at the moment as interaction with Fedora is done through localhost and camel connects through the docker hostname (fcrepo). This leads to camel toolbox failing to connect to Fedora as it appears to be getting the url to invoke from the http request.

# Interested parties
@fcrepo/committers
